swagger.json

```diff
@@ -1121,6 +1121,122 @@
+                      "in": "path",
+                      "name": "driver_id",
+                      "required": true,
+                      "type": "integer"
+                    },
+                    {
+                        "description": "A specified timestamp within the bounds of a driver's certified daily log, specified in milliseconds UNIX time.",
+                        "in": "path",
+                        "name": "time_ms",
+                        "required": true,
+                        "type": "integer"
+                    }
+                ],
+                "responses": {
+                    "200": {
+                        "description": "Returns a certified daily log for the driver, if one exists",
+                        "schema": {
+                            "$ref": "#/definitions/HosCertifiedDailyLogResponse"
+                        }
+                    },
+                    "default": {
+                        "description": "Unexpected error.",
+                        "schema": {
+                            "$ref": "#/definitions/ErrorResponse"
+                        }
+                    }
+                }
+            },
+            "patch": {
+              "summary": "/fleet/drivers/{driver_id:[0-9]+}/hos_certified_daily_logs/{time_ms:[0-9]+}",
+              "description": "Modify details, such as trailer number or shipping ID, for a driver's certified daily log",
+              "operationId": "patch_hos_certified_daily_log",
+              "tags": [
+                "Fleet", "Drivers"
+              ],
+              "produces": [
+                "application/json"
+              ],
+              "consumes": [
+                "application/json"
+              ],
+              "parameters": [
+                {
+                  "$ref": "#/parameters/accessTokenParam"
+                },
+                {
+                  "description": "ID (integer) of the driver with certified daily log.",
+                  "in": "path",
+                  "name": "driver_id",
+                  "required": true,
+                  "type": "integer"
+                },
+                {
+                    "description": "A specified timestamp within the bounds of a driver's certified daily log, specified in milliseconds UNIX time.",
+                    "in": "path",
+                    "name": "time_ms",
+                    "required": true,
+                    "type": "integer"
+                },
+                {
+                  "in": "body",
+                  "name": "hosCertifiedShiftParams",
+                  "required": true,
+                  "schema": {
+                    "properties": {
+                      "shippingDocIds": {
+                        "description": "One or more customer shipping document IDs to add to driver's daily certified log. Multiple numbers can be inserted, delimited by a comma or space. Max of 255 characters",
```

Comment:
Do we really want to offer space separated value support? Can we just assert that multiple values have to be comma separated?